src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/onetimecode/OneTimeCodeService.kt

```diff
@@ -1,17 +1,42 @@
 package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.onetimecode
 
 import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.cjsm.CjsmService
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.ResourceNotFoundException
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.security.JwtService
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.security.SmokeTestConfig
 
 @Service
 class OneTimeCodeService(
   private val oneTimeCodeGenerator: OneTimeCodeGenerator,
   private val oneTimeCodeRepository: OneTimeCodeRepository,
   private val oneTimeCodeEmailSender: OneTimeCodeEmailSender,
+  private val jwtService: JwtService,
+  private val cjsmService: CjsmService,
+  private val smokeTestConfig: SmokeTestConfig,
```

Comment:
@nathanrussell-moj-digital  - It's a bit strange to have a test config on a non-test file?